.github/CODEOWNERS

```
@@ -342,6 +342,7 @@
 /packages/redis @elastic/obs-infraobs-integrations
 /packages/redisenterprise @elastic/obs-infraobs-integrations
 /packages/rubrik @elastic/obs-infraobs-integrations
+/packages/sailpoint_identity_security_cloud @elastic/security-service-integrations
```

```suggestion
/packages/sailpoint_identity_sc @elastic/security-service-integrations
```

We should also change the package directory name to reflect the same.

---

this change will require you to regen the tests as the data stream names will change @HeroicHorizon